src/PhpSpec/Console/Provider/NamespacesAutocompleteProvider.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Console\Provider;
+
+use Symfony\Component\Finder\Finder;
+
+final class NamespacesAutocompleteProvider
+{
+    /**
+     * @var Finder
+     */
+    private $finder;
+
+    public function __construct(Finder $finder)
+    {
+        $this->finder = $finder;
+    }
+
+    /**
+     * Get namespaces from paths.
+     *
+     * @param  array $paths
+     *
+     * @return array of namespases
+     */
+    public function getNamespaces($paths)
+    {
+        $namespaces = array();
```

Comment:
Since `phpspec` requires PHP 5.6 or higher it makes sense to use short array syntax in new code.